### ISSUE TITLE
review homepage a bit?

### ISSUE THREAD

**Issue by maelle** [ropensci/roweb3#573]
@jeroen @yabellini @mpadge 

I was looking at https://ropensci.org/ and wondering whether the content of the homepage (not only the navbar) should mention other projects (R-universe, champions, stat software review) a bit more prominently.

**Comment by mpadge** [comment]
Yeah, that's a good idea. "Packages" is both in the navbar and the first main page div, but "projects" only gets a navbar entry and none appear anywhere on main page. An additional "Projects" div?

**Comment by maelle** [comment]
later we could add multilingual publishing as well
